runtime/local/Cargo.toml

```
@@ -43,11 +43,13 @@ pallet-insecure-randomness-collective-flip = { workspace = true }
 pallet-membership = { workspace = true }
 pallet-preimage = { workspace = true }
 pallet-proxy = { workspace = true }
+pallet-safe-mode = { workspace = true, default-features = false }
```

defined on workspace 
```suggestion
pallet-safe-mode = { workspace = true }
```